### ISSUE TITLE
WCAG 2.0: A customizable quick  reference draft

### ISSUE THREAD

**Issue by zbolena** [w3c/wai-quick-start#306]
The newly drafted WCAG  quick reference site is considerably more usable by the addition of filtering and layout. 

**Share this view:** 
The option to “share this view” is very helpful to send specifically filtered content. 

Though I would like to just share the specifically filtered content (guidelines) without the rest of the surrounding left-hand sidebar content (content/filter column, frame header) so not confuse the user I am sharing the content with. 

**Link to Understanding (guideline):**
Would be nice to have this content collapsed within this page instead of a separate link. Though I can understand it is a considerable amount of content for each guideline.

**Filtering:** 
I really like the tag filtering mechanism. I think this really helps user to quickly view the specifically related content needed.

**Custom filters:** 
Lately I have been assembling docs for our development teams with specific topics such as state, progress, status, for navigation, tabs, accordions, carousels, etc. It would be nice to be able to enter a custom filter for specific content.

The WCAG site is very comprehensive and contains an abundance of content that can be overwhelming to many. A filtering mechanism like this would be a wonderful addition to the site.

This would be great to get material for specific topics such as "state, progress, status" relating to  and retrieve the related guidelines, success techniques, failures, patterns, authoring practices, relationships, etc.

**Comment by yatil** [comment]
Moved to the correct repository. Sorry for the confusion. You can find it here: w3c/wai-wcag-quickref#96.
